lucet-runtime/lucet-runtime-internals/src/vmctx.rs

```diff
@@ -232,14 +237,18 @@ impl Vmctx {
     /// This is almost certainly not what you want to use to terminate an instance from a hostcall,
     /// as any resources currently in scope will not be dropped. Instead, use
     /// `lucet_hostcall_terminate!` which unwinds to the enclosing hostcall body.
-    pub unsafe fn terminate_no_unwind(&mut self, details: TerminationDetails) -> ! {
+    pub unsafe fn terminate_no_unwind(&self, details: TerminationDetails) -> ! {
```

Comment:
No check is done here, because this is already `unsafe` and only used in contexts where there are no borrows yet, like the expansion of the `#[lucet_hostcall]` macro.